### DISCUSSION TITLE
Not working: bind: cannot assign requested address

### DISCUSSION THREAD

**Discussion by gugu-hw** [v2rayA/v2rayA#1469] [Q&A]
OS: openSUSE Tumbleweed 20240807
Kernel: 6.10.3-1-default
V2rayA version: 2.2.5.8
Core version: Xray 1.8.23
Logs:
2024/08/09 16:40:08.293 [I] [io.go:431] Xray 1.8.23 (Xray, Penetrates Everything.) 4c82ef8 (go1.22.5 linux/amd64)
2024/08/09 16:40:08.293 [I] [io.go:431] A unified platform for anti-censorship.
2024/08/09 16:40:08.293 [I] [io.go:431] [Info] infra/conf/serial: Reading config: /etc/v2raya/config.json
2024/08/09 16:40:08.375 [I] [io.go:431] Failed to start: app/proxyman/inbound: failed to listen TCP on 20171 > transport/internet: failed to listen on address: [::1]:20171 > transport/internet/tcp: failed to listen TCP on [::1]:20171 > listen tcp [::1]:20171: bind: cannot assign requested address
2024/08/09 16:40:08.397 [W] [transparent.go:74] writeTransparentProxyRules: not support "system proxy" mode of transparent proxy: does not support to configure system proxy on your OS
2024/08/09 16:40:08.494 [E] [connection.go:88] failed to start v2ray-core: not support "system proxy" mode of transparent proxy: does not support to configure system proxy on your OS
#1466 Seemes the same error, may meaning that not a core bug? (He's using v2ray core and I'm using Xray)

**Comment by eronez** [comment]
1.
```shell
ip a
```
whether addredd "::1"  exists or not

2.
```shell
sudo lsof -i:20171
```
check this port is occupied or not

3.
"not support "system proxy" mode of transparent proxy"
try other mode
